### DISCUSSION TITLE
Car shadows as in the Lindbergh version

### DISCUSSION THREAD

**Discussion by Telmator** [emoose/OutRun2006Tweaks#251] [Suggestions]
### Describe the feature you'd like to see added

Hello!

I wonder if it would be possible to add shadows to the rest of the cars, not just the player car. That would be a nice cosmetic addition, I noticed that the Lindbergh version of the game has that.

Thanks!

**Comment by Outrunner24** [comment]
> I wonder if it would be possible to add shadows to the rest of the cars

They already have shadows, what you're talking about are STENCIL shadows, the projections that change with the cars' or camera's position. 

![sten](https://github.com/user-attachments/assets/08fe70a1-c3c2-415b-9bbf-1d3b3d6f6ad4)
_(lindbergh screenshot from youtube)_

Those require changes to the graphics engine and I don't think it's possible unless there's a way of porting the code from the arcade original to the PC and mix things up.

**Comment by Blackbird88** [comment]
OutRun Online Arcade has them too. I wonder if they're just disabled for traffic since player car has it
![image](https://github.com/user-attachments/assets/02824f5a-9fa2-4966-9b94-c15425554499)

**Comment by Outrunner24** [comment]
> OutRun Online Arcade has them too. I wonder if they're just disabled for traffic since player car has it 

Interesting, looks like the devs disabled them for the older PC release (for better performance) then they were put back in for the later ports. That said I doubt they can be modded into the PC version without access to the game code, unless there's a flag somewhere that enables/disables this feature.

**Reply by SecularSteve** [reply]
I hope there is.
It's one of those "can't unsee it once you do" things.
